app/v1/controllers/SubmitReturnController.scala

```diff
@@ -67,32 +66,25 @@ class SubmitReturnController @Inject()(val authService: EnrolmentsAuthService,
 
       val result = for {
         parsedRequest <- EitherT.fromEither[Future](requestParser.parseRequest(rawRequest))
-        nrsResponse <- EitherT(nrsService.submitNrs(parsedRequest, submissionTimestamp))
+        nrsSubmissionId <- EitherT.right(nrsService.submitNrs(parsedRequest, submissionTimestamp))
```

Comment:
This won't be async just yet - it will still rely on the result of this call coming back.  Let me throw up a PR quickly that fixes this

Reply:
ok thanks